cypress/features/regression/experimental/textarea.feature

```diff
@@ -0,0 +1,272 @@
+  Scenario Outline: Set cols to <cols>
+    When I set cols slider to <cols>
+    Then cols is set to "<cols>"
+    Examples:
+      | cols |
+      | 1    |
+      | 115  |
+      | 299  |
+      | 300  |
+  
+  @positive
+  Scenario Outline: Set rows to <rows>
+    When I set rows slider to <rows>
+    Then rows is set to "<rows>"
+    Examples:
+      | rows |
+      | 1    |
+      | 115  |
+      | 299  |
+      | 300  |
+ 
+  @positive
+  Scenario: Check disabled checkbox for a Textarea component
+    When I check disabled checkbox
+    Then Textarea component is disabled
+
+  @positive
+  Scenario: Uncheck disabled checkbox for a Textarea component
+    When I check disabled checkbox
+      And I uncheck disabled checkbox
+    Then Textarea component is not disabled
+
+  @positive
+  Scenario: Enable readOnly checkbox for a Textarea component
+    When I check readOnly checkbox
+    Then Textarea component is readOnly
+
+  @positive
+  Scenario: Disable readOnly checkbox for a Textarea component
+    When I check readOnly checkbox
+      And I uncheck readOnly checkbox
+    Then Textarea component is not readOnly
+ 
+  @positive
+  Scenario Outline: Set placeholder to <placeholder>
+    When I set placeholder to "<placeholder>"
+    Then placeholder is set to "<placeholder>"
+    Examples:
+      | placeholder             |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <> |
+
+  @positive
+  Scenario Outline: Set fieldHelp to <fieldHelp>
+    When I set fieldHelp to "<fieldHelp>"
+    Then fieldHelp is set to "<fieldHelp>"
+    Examples:
+      | fieldHelp               |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <> |
+
+  @positive
+  Scenario Outline: Set characterLimit to <characterLimit>
+    When I set characterLimit to "<characterLimit>"
+    Then characterLimit is set to "<characterLimit>"
+      And characterLimit for default Textarea is shown as "<result>"
+    Examples:
+      | characterLimit | result  |
+      | -1000          | -1,000  |
+      | -1             | -1      |
+      | 0              | 0       |
+      | 1              | 1       |
+      | 100            | 100     |
+      | 1000           | 1,000   |
+      | 555555         | 555,555 |
+
+  @negative
+  Scenario Outline: Set characterLimit out of scope to <characterLimit>
+    When I set characterLimit to "<characterLimit>"
+    Then characterLimit for default Textarea is not set to "<characterLimit>"
+    Examples:
+      | characterLimit          |
+      | Sample text             |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+      | -1                      |
```

Comment:
Are you sure that `-1` as value is not shown?

Reply:
fixed. I removed it